requirements.in

```diff
@@ -24,7 +24,7 @@ dagster-celery==0.25.8
 # dagster-cloud==1.9.8
 deltalake==0.23.2
 dj-database-url==0.5.0
-Django~=4.2.15
```

Comment:
you also need to "compile" this for it to take effect

i believe the command (if we didn't change a workflow) is

`uv pip compile requirements.in -o requirements.txt && uv pip compile requirements-dev.in -o requirements-dev.txt
`